enterprise_subsidy/apps/api/v1/tests/test_views.py

```diff
@@ -259,46 +260,40 @@ def test_create_new_subsidy_with_permissions(self, role, status_code):
         elif role == "operator":
             self.set_up_operator()
         url = self.get_list_url
-        data = {
-            "reference_id": "aksdkjtkwekwl88890",
-            "default_title": "title",
-            "default_enterprise_customer_uuid": str(uuid.uuid4()),
-            "default_unit": UnitChoices.USD_CENTS,
-            "default_starting_balance": "10000",
-            "default_revenue_category": RevenueCategoryChoices.BULK_ENROLLMENT_PREPAY,
-            "default_internal_only": True,
-        }
+        factory_data = build(dict, FACTORY_CLASS=SubsidyFactory)
```

Comment:
Never seen this before, is this different than SubsidyFactory.build()?

Reply:
I think they're basically the same, except SubsidyFactory.build().__dict__ will have a "_state" key, and with build(dict, FACTORY_CLASS=SubsidyFactory) I didn't have to take the extra step to remove "_state"